### PR TITLE
Getting the binaries from S3 & setting latests

### PR DESCRIPTION
**Pull request by shanimal08**
## Description

- Getting the libs data from S3 and rendering it (only for Libraries at the moment)
- Setting the download url and checksums for `latest` for everything but Libraries

#### Screenshots

It looks the same but... :)

image

image

image

### CODE REVIEW

File: _includes/downloads.html [hunk 1]
```diff
@@ -70,18 +70,20 @@ <h2 class="dl-title">{{ os.title }}</h2>
                     {% endif %}
                     {% if os.checksum %}
                     <div class="checksum">
-                        SHA256: {{ os.checksum }} 
-                        
+                        SHA256: {{ os.checksum }}
```
Comment (Krakaw) [inline _includes/downloads.html]: This changes with every file, it needs to come from the API

**Review by Krakaw — CHANGES_REQUESTED** [review]

File: _includes/downloads.html [hunk 2]
```diff
@@ -35,12 +35,12 @@ <h2 class="dl-title">{{ os.title }}</h2>
...
                         For {{ os.version }}
-                       
+
                     </div>
                     {% endif %}
 
-                    {% if os.download %}
-                    <a class="btn " href="{{ site.baseurl}}/{{ os.download }}">{{os.ctaText}}</a>
+                    {% if os.downloadBtnID %}
+                    <a class="btn" id="{{ os.downloadBtnID }}">{{os.ctaText}}</a>
```
Comment (Krakaw) [inline _includes/downloads.html:43]: Where's the href?

**Review by Krakaw — CHANGES_REQUESTED** [review]

File: _includes/downloads.html (hunk 2 above)
Comment (shanimal08) [reply _includes/downloads.html:43]: I set it in the JS on line 81 😬

**Review by Krakaw — APPROVED** [review]
LGTM!

**Review by CjS77 — APPROVED** [review]
Nice

**Pull request merged**